code/modules/projectiles/guns/shotguns.dm

```diff
@@ -325,9 +325,10 @@
 	dry_fire_sound = 'sound/weapons/guns/fire/sniper_empty.ogg'
 	reload_sound = 'sound/weapons/guns/interact/mosin_reload.ogg'
 	caliber = CALIBER_762X54 //codex
-	load_method = SINGLE_CASING //codex
-	max_chamber_items = 4 //codex
+	load_method = SINGLE_CASING|SPEEDLOADER //codex
+	max_chamber_items = 5 //codex
```

Comment:
why change ammo counts

Reply:
Not tested since. I'll do it tomorrow.

Reply:
> why change ammo counts
This doesn't change the amount of bullets the gun holds.  It changes the amount of Bullets you can load into the gun.  The mosin held 4 and then you'd have to 'pump' the fourth round into the chamber then load the last fifth bullet into the gun.

I asked about this jank on this Dev General, and a user named Bird told me a way to fix that jank.  That convo can be found via this link https://discord.com/channels/498569646014857217/500801682058379265/958504597813624912

Reply:
It appears the gun CAN hold six bullets if you load the last one without using the stripper clip (using a handful). This wouldn't be the case if we changed something.. else. Perhaps internal magazine, if it is because of how that works.
Otherwisedly the stripper clips work fine now. @PopeWreckoning Reminder that you are not branched inside your fork so this makes your master not loadable as a branch, I've had to code the commit "allowed_ammo_types" for myself.

Reply:
I thought this would be an easy fix only to find in order to prevent this plus one issue to my scouring of the code is to fuck with the /obj/item/weapon/gun, so I'm not going to be touching that with a 20ft pole even assuming what i think is correct.  I've tried every conceivable method I thought to the variables to try and make it 5 rounds, but due to the Receiver flag having AMMO_RECIEVER_REQUIRES_UNIQUE_ACTION, and some sort of code fuckery that carries i cant seem to fix it with a variable change to the mosins code while Also have the stripper clips be Five rounds.

So, fuck it 5+1 is just going to be the Mosin with this Pr.  If that is a problem then i hope you know how to fix it because i havent an idea other than lowering the clips to 4 rounds and that be the end.

People in dev-general in the discord is saying this is fine since its not a martini damage rifle.